ops/testing.py

```diff
@@ -3285,9 +3254,9 @@ def send_signal(self, sig: Union[int, str], service_names: Iterable[str]):
             # conform with the real pebble api
             first_service = next(iter(service_names))
             message = f'cannot send signal to "{first_service}": invalid signal name "{sig}"'
-            raise self._api_error(500, message)
+            raise self._api_error(500, message) from None
 
-    def get_checks(self, level=None, names=None):  # type:ignore
+    def get_checks(self, level=None, names=None):  # type:ignore  # noqa: ANN001
```

Comment:
For ANN001 can we add type annotations, seeing we're using Pyright on everything anyway?